### PR TITLE
[#167270217] Min app version support in API

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Changes were made to package.json, but not to yarn.lock.<br/><i>Perhaps you need to run `yarn install`?</i>
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#167270217](https://www.pivotaltracker.com/story/show/167270217): Il backend deve dichiarare la versione minima dell'app con cui è compatibile

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/app.ts [hunk 1]
```diff
@@ -537,8 +537,14 @@ function registerPublicRoutes(app: Express): void {
   const packageJson = require("../package.json");
   const version = t.string.decode(packageJson.version).getOrElse("UNKNOWN");
 
+  // The minimum app version that support this API
+  const minAppVersion = t.string
+    .decode(packageJson.minAppVersion)
+    .getOrElse("0.0.0");
```
Comment (cloudify) [inline src/app.ts]: this is a lie! 😂 it's probably better to reply with `UNKNOWN` and let the client decide what to do

File: package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "italia-backend",
   "version": "1.6.2",
+  "minAppVersion": "0.0.0",
```
Comment (cloudify) [inline package.json]: also add some docs to the `README` (I like your choice of storing the info in package.json)

Comment (BurnedMarshal) [reply package.json]: fixed in 2909928

File: src/app.ts (hunk 1 above)
Comment (BurnedMarshal) [reply src/app.ts]: fixed in 2909928

File: README.md [hunk 3]
```diff
@@ -183,6 +184,10 @@ Redis Database stores data required only by application side functionalities. Be
 | SESSIONINFO-HexToken   | a JSON representing the `SessionInfo` object | `SessionInfo` | TOKEN_DURATION_IN_SECONDS |
 | USERSESSIONS-MRARSS80A01H501T | a Set of SessionInfo Keys | `Set<SessionInfoKey>` | never |
 
+## Mobile App compatibility
+
+To handle Backend compatibility with several Mobile App versions, the oldest mobile app version supported by the backend is stored into the property `minAppVersion` inside the `package.json`. This value is provided to the app throw the `/info` API.
```
Comment (cloudify) [inline README.md]: throw -> through

Comment (BurnedMarshal) [reply README.md]: fixed in 2039489 😅

**Pull request merged**